plugins/woocommerce-admin/client/customize-store/assembler-hub/hooks/use-pattern.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @woocommerce/dependency-group */
```

Comment:
There is a same hook:  [`plugins/woocommerce-admin/client/customize-store/assembler-hub/hooks/use-patterns.ts`](https://github.com/woocommerce/woocommerce/blob/378273d6270339ba5f9e07d1817f33fa909e99ca/plugins/woocommerce-admin/client/customize-store/assembler-hub/hooks/use-patterns.ts). Should we use it? 😅

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/40136/commits/4b90b64e70a5ab3701235f1be27fe0ec6aae037d